lumigator/python/mzai/backend/backend/services/jobs.py

```diff
@@ -7,7 +7,8 @@
 from schemas.extras import ListingResponse
 from schemas.jobs import (
     JobConfig,
-    JobCreate,
+    JobEvalCreate,
```

Comment:
This is to accomodate different input schemas for different input jobs

Reply:
Thanks Irina! My reasoning is that, since we are removing inference from the evaluation job (see Vicki's [draft PR](https://github.com/mozilla-ai/lumigator/pull/333)), we won't have such a big overlap anymore.
I agree on factoring as much code together as possible though and, despite preferring to keep different, more specific pydantic configs (until we actually see the need to group stuff together because there is a non-trivial overlap), I think we could do better in terms of factoring out commonalities in the code